front/app/containers/ProjectsShowPage/shared/header/ProjectActionButtons.tsx

```diff
@@ -148,29 +155,41 @@ const ProjectActionButtons = memo<Props>(({ projectId, className }) => {
     project,
     phases
   );
+
   const isParticipationMethodIdeation = participation_method === 'ideation';
+
   const isParticipationMethodNativeSurvey =
-    participation_method === 'native_survey';
+    participation_method === 'native_survey' ||
+    currentPhase?.attributes.participation_method === 'native_survey';
+
```

Comment:
The existing logic in the ProjectActionButtons file is pretty messy, but I think it will take a decent chunk of time to rework it. Maybe better to leave it like this for now but add it on our to-do list?